git/commitizen.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  types: [
+    { value: 'feat', name: 'feat:      Добавление нового функционала' },
+    { value: 'fix', name: 'fix:       Исправление ошибок' },
+    { value: 'test', name: 'test:      Добавление тестов' },
+    {
+      value: 'refactor',
+      name: 'refactor:  Правки кода без исправления ошибок или добавления новых функций',
+    },
+    { value: 'docs', name: 'docs:      Обновление документации' },
+    { value: 'chore', name: 'chore      Добавление/обновление/настройка инструментов и библиотек' },
+  ],
+
+  scopes: [{ name: 'custom' }, { name: 'components' }, { name: 'global' }],
+
+  messages: {
+    type: 'Тип изменений',
+    scope: 'Область изменений (опционально):',
+    customScope: 'Введите вашу область:',
+    subject: 'Краткое описание изменений:\n',
+    body: 'Подробное описание изменений:\n',
+    breaking: 'Описание BREAKING CHANGES для повышения мажорной версии пакета:\n',
```

Comment:
наверное стоит сформулировать так
```suggestion
    breaking: 'Описание BREAKING CHANGES:\n',
```

этот конфиг может много где использоваться, не обязательно в либах или пакетах которые будут публиковаться
значит там не обязательно будет версионирование согласно спеке conventional commits